fcrepo-api-x-model/src/main/resources/ontologies/apix-extension.ttl

```diff
@@ -0,0 +1,65 @@
+@prefix apix:<http://fedora.info/definitions/v4/api-extension#> .
+@prefix fcr:<http://fedora.info/definitions/v4/repository#> .
+@prefix owl:<http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:<http://www.w3.org/2000/01/rdf-schema#> .
+@prefix svc:<http://fedora.info/definitions/v4/service#> .
+@prefix xsd:<http://www.w3.org/2001/XMLSchema#> .
+
+apix: a owl:Ontology;
+    owl:imports svc:;
+    rdfs:label "API Extension ontology" .
+
+apix:Extension a owl:Class;
+    rdfs:label "API-X extension"@en;
+    rdfs:comment
+    "Relates Services to Repository objects. May expose a service on repository resources (‘exposing’ modality), or modify request or responses to repository objects (‘intercepting’ modality)."@en .
```

Comment:
Yeah google does it too.  It's bitten me multiple times!

On Tue, Aug 30, 2016 at 3:33 PM, birkland <notifications@github.com> wrote:

> In fcrepo-api-x-model/src/main/resources/ontologies/apix-extension.ttl
> <https://github.com/fcrepo4-labs/fcrepo-api-x/pull/38#discussion_r76863849>
> :
>
> > @@ -0,0 +1,65 @@
> > +@prefix apix:<http://fedora.info/definitions/v4/api-extension#> .
> > +@prefix fcr:<http://fedora.info/definitions/v4/repository#> .
> > +@prefix owl:<http://www.w3.org/2002/07/owl#> .
> > +@prefix rdfs:<http://www.w3.org/2000/01/rdf-schema#> .
> > +@prefix svc:<http://fedora.info/definitions/v4/service#> .
> > +@prefix xsd:<http://www.w3.org/2001/XMLSchema#> .
> > +
> > +apix: a owl:Ontology;
> > +    owl:imports svc:;
> > +    rdfs:label "API Extension ontology" .
> > +
> > +apix:Extension a owl:Class;
> > +    rdfs:label "API-X extension"@en;
> > +    rdfs:comment
> > +    "Relates Services to Repository objects. May expose a service on repository resources (‘exposing’ modality), or modify request or responses to repository objects (‘intercepting’ modality)."@en .
>
> Ha, That was a cut 'n paste from the rendered markdown - I didn't realize
> it did that.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fcrepo4-labs/fcrepo-api-x/pull/38/files/29c401b3651622a342cdfd6ca7183566ce11d612#r76863849>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAI-Gg1vw3H-BbISUBwRa35EU0cy3XKQks5qlIWNgaJpZM4Jw3ji>
> .
>